app/dashboard/widgets/access_logs.php

```diff
@@ -35,7 +35,7 @@
 	$logs = $Log->fetch_logs($max, null, null, null, "on", "off", "off");
 	if (!is_array($logs)) { $logs = array(); }
 
-	print "<div" . (isset($height) ? " style=\"height:{$height}px;overflow:scroll;width:98%;margin-left:1%;\"" : "") . ">";
+	print '<div style="width:98%;margin-left:1%;' . (isset($height) ? "height:{$height}px;overflow-y:auto;" : "") . '">';
```

Comment:
Hi, this breaks / intruduces padding on widgets (width:98%,margin-left:1%)

Reply:
Correct, this was intentional. Some of the widgets had this, like the Statistics widget, and I thought it would be good if the widgets had a uniform appearance. Do you dislike it?